### ISSUE TITLE
Remove trailing slashes from filename directory argument

### ISSUE THREAD

**Issue by verhovsky** [markdownlint/markdownlint#495]
## Description
I did this:

```sh
mdl source/docs/
```
### Environment
macos

**MDL Version**
0.13.0
### Expected Behavior

```
source/docs/awk.html.markdown:6: MD032 Lists should be surrounded by blank lines
source/docs/awk.html.markdown:6: MD034 Bare URL used
source/docs/ballerina.html.markdown:427: MD002 First header should be a top level header
source/docs/ballerina.html.markdown:8: MD009 Trailing spaces
source/docs/ballerina.html.markdown:13: MD009 Trailing spaces
...
```
### Actual Behavior
```
source/docs//awk.html.markdown:6: MD032 Lists should be surrounded by blank lines
source/docs//awk.html.markdown:6: MD034 Bare URL used
source/docs//ballerina.html.markdown:427: MD002 First header should be a top level header
source/docs//ballerina.html.markdown:8: MD009 Trailing spaces
source/docs//ballerina.html.markdown:13: MD009 Trailing spaces
...
```

there are two slashes
